mne_bids/tests/test_write.py

```diff
@@ -401,7 +401,12 @@ def test_fif(_bids_validate, tmpdir):
     ch_pos = dict(zip(ch_names, elec_locs))
     meg_montage = mne.channels.make_dig_montage(ch_pos=ch_pos,
                                                 coord_frame='head')
-    raw.set_montage(meg_montage)
+
+    if check_version('mne', '0.24'):
+        with pytest.warns(RuntimeWarning, match='Not setting position'):
+            raw.set_montage(meg_montage)
+    else:
+        raw.set_montage(meg_montage)
```

Comment:
My additional commit made the comment from https://github.com/mne-tools/mne-bids/pull/807#discussion_r650887250 disappear here, but it's still important.